pkgs/games/keeperrl/default.nix

```diff
@@ -0,0 +1,83 @@
+{ stdenv, fetchFromGitHub, makeWrapper, requireFile
+, openal, curl, libogg, libvorbis
+, SDL2, SDL2_image, zlib
+, unfree_assets ? false }:
+
+with stdenv.lib;
+
+let
+  pname = "keeperrl";
```

Comment:
looks like this isn't used anywhere, however, i think most people prefer the use of pname over the explicit use of the name in general.